static/app/views/dashboards/widgetBuilder/components/datasetSelector.tsx

```diff
@@ -79,8 +74,3 @@ const FeatureBadgeAlignmentWrapper = styled('div')`
     top: -1px;
   }
 `;
-
-const Header = styled('h6')`
-  font-size: ${p => p.theme.fontSizeLarge};
-  margin-bottom: ${space(2)};
```

Comment:
are we able to keep this different bottom margin for the dataset header? it looks a little better and closer to the design 👀

Reply:
Ohh I didn't realize it was a bit more! hmmm let's see

Reply:
I added a way to apply styles using the styled component for this. It's a bit hacky but for now I don't think it needs to change much

Reply:
amazing thanks!